apps/okreads-e2e/src/specs/search-books.spec.ts

```diff
@@ -16,12 +16,17 @@ describe('When: Use the search feature', () => {
     expect(items.length).toBeGreaterThan(1);
   });
 
-  xit('Then: I should see search results as I am typing', async () => {
+  it('Then: I should see search results as I am typing', async () => {
```

Comment:
This is actually part of the next task for instant search feature so you don't need it as part of this task. :)